grc/templates/upload/document-password.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'base.html' %}
+{% import "govuk-design-system-templates/error-summary.html" as error_summary %}
+{% import "govuk-design-system-templates/text-input.html" as textInput %}
+{% import "upload/_upload-files-section.html" as upload_files_section %}
+
+{% block title %}Enter your document password{% endblock %}
+
+{% block backLink %}
+    <a href="{{ url_for('upload.uploadInfoPage', section_url=section_url) }}" class="govuk-back-link">Back</a>
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            {{ error_summary.renderFor(passwordForm) }}
+
+            <h1 class="govuk-heading-l">
+                Enter your document {% if currently_uploaded_files | length == 1 %}password{% else %}passwords{% endif %}
```

Comment:
Can you do a demo (from your laptop) to Spence/Tor/etc so they can check this workflow.
If there's more than one password-protected file, it visually looks like you can enter all the passwords in one go.
But if you do, it only accepts one password and ignores all the others.